client/src/CodeClaimPage/QRHashForm.tsx

```diff
@@ -69,6 +74,26 @@ const QRHashForm: React.FC<{error: boolean, loading: boolean, checkClaim: (hash:
         </Formik>
       </div>
       <ClaimFooterMessage />
+
+      <ReactModal isOpen={isModalOpen} shouldFocusAfterRender={true}>
+        <div className="admin-list-modal">
+          <div className='claim-modal-text-container'>
+            <p>The current surge in gas prices makes a POAP minting have a cost of between $2 and $6 in mining fees.</p>
+            <p>We are working hard on a scaling solution that should be ready in 6-8 weeks. </p>
+            <p>In the meantime please input your address in the field below and click on claim. 
+              That’s going to block this POAP to be minted for free once our new deployment is ready. 
+              Afterwards you can continue the process at your expense or just close this page. 
+              For learning more about what our plans are visit our discord. 
+            </p>
+            <p><a href="http://poap.xyz/discord" target="_blank" rel="noopener noreferrer">http://poap.xyz/discord</a></p>
+          </div>
+          <div className="claim-modal-cancel-container">
+            <div onClick={() => handleModal()} className={'close-modal'}>
```

Comment:
Prefer avoiding the double arrow functions, this is what you are writing:

```javascript
            <div onClick={() => (() => setIsModalOpen(false))()} className={'close-modal'}>
```

This is what we want to write and it translates 
```javascript
            <div onClick={handleModal} className={'close-modal'}>
```